### ISSUE TITLE
Can't use transformations with glide 4.0 RC

### ISSUE THREAD

**Issue by monajafi** [wasabeef/glide-transformations#72]
I tried to use  custom transformations in glide 4.0 RC but it didn't applied. here is the code sample I tried:
```
RequestOptions options = new RequestOptions();
                        options.bitmapTransform(new CropCircleTransformation(mContext));
                        Glide.with(mContext).load(item.getParseFile("image").getUrl()).apply(options)
                                .into(vh.imgView);
```

**Comment by AllanWang** [comment]
I just use [Circle Crop](https://github.com/bumptech/glide/blob/master/library/src/main/java/com/bumptech/glide/load/resource/bitmap/CircleCrop.java), which is directly in glide

**Comment by AndSync** [comment]
copy the source code into your project and implement the new abstract method

**Comment by wasabeef** [comment]
I just released glide-transformations 3.0.0. So could you check that?
